src/browser/workspace-api.ts

```diff
@@ -45,39 +40,20 @@ export class RestDevWorkspaceApi implements IDevWorkspaceApi {
     workspaceName: string
   ): Promise<IDevWorkspace> {
     const resp = await this.axios.get(
-      `/apis/${devWorkspaceApiGroup}/${devworkspaceVersion}/namespaces/${namespace}/${devworkspaceSubresource}/${workspaceName}`
+      `/apis/${devWorkspaceApiGroup}/${devworkspaceVersion}/namespaces/${namespace}/${devworkspacePluralSubresource}/${workspaceName}`
     );
     return resp.data;
   }
 
   async create(
     devfile: IDevWorkspaceDevfile,
-    defaultEditor?: string,
-    defaultPlugins?: string[]
+    started = true
```

Comment:
I assume routingClass should be parameter as well if we want this code be Che independent, but that's more comment for future refactoring of that Library.